test-suite/js/src/test/scala/org/scalajs/testsuite/scalalib/ScalaRunTimeJSTest.scala

```diff
@@ -0,0 +1,22 @@
+package org.scalajs.testsuite.scalalib
+
+import org.junit.Test
+import org.junit.Assert.assertFalse
+
+import scala.scalajs.js
+
+class ScalaRunTimeJSTest {
+
+  @Test def ScalaRunTime_isArray_should_not_fail_with_JS_objects(): Unit = {
+    def isScalaArray(x: Any): Boolean =
+      x match {
+        case _: Array[t] => true
+        case _ => false
+      }
```

Comment:
Coding style:
* Either start `x match {` on the right of the `=`, or you need an additional pair of `{}` around the entire body.
* Vertically align the `=>` of a `match` if `case`s are one-liners.
* `t` is unused and so it should be `_`

More information: https://github.com/scala-js/scala-js/blob/master/CODINGSTYLE.md